scripts/docker/content/set_env.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
```

Comment:
My comments for this file are same as what I reviewed for webui [PR](https://github.com/Azure/pcs-remote-monitoring-webui/pull/1235)